tests/Makefile

```diff
@@ -1,4 +1,4 @@
-COQC=coqc
+COQC=coqc -Q ../theories Hammer -Q ../src Hammer.Plugin -I ../src
```

Comment:
I think a major point of having the tests is to also check if things got installed correctly (there was a bug in this e.g. in one of your previous commits). So the tests should be run after the installation without using -Q and -I (besides, it won't work before installation for the plugin, because `predict` and `htimeout` are not in path).

Reply:
Maybe also "make clean" before the tests?

Reply:
I see your point, but the testing options in OPAM are tailored for testing _builds_. I'll see if I can come up with something.